COLLADAStreamWriter/include/COLLADASWConstants.h

```diff
@@ -202,6 +202,8 @@ namespace COLLADASW
         static const String CSW_ELEMENT_PASS;
         static const String CSW_ELEMENT_PH;
         static const String CSW_ELEMENT_PHONG;
+		static const String CSW_ELEMENT_PLUGIN_VERSION;
```

Comment:
CSW_ELEMENT_PLUGIN_VERSION = "pluginVersion" : the element in the DAE with the pluginVersion:
CURRENT_REVISION = "developer version"   >> patched by jenkins
<pluginVersion> CURRENT_REVISION  </pluginVersion>
<SHA1> 4549f95 </SHA1> or no element if build by jenkins